src/lang.js

```diff
@@ -37,8 +37,8 @@ var supportedLanguages = [
     ["sv", "SV"], // Swedish
     ["tr", "TR"], // Turkish
     ["uk", "UK"], // Ukrainian
-    ["zh-Hans", "ZH"], // Simplified Chinese
-    ["zh-Hant", "ZH"], // Traditional Chinese
+    ["zh-Hans", "ZH-HANS"], // Simplified Chinese
```

Comment:
Should be `zh-Hans` and `zh-Hant`

Reply:
https://developers.deepl.com/docs/resources/supported-languages#target-languages

Reference the page.

Reply:
OK, Thanks. I noticed in the packet capture that `zh-Hans` and `zh-Hant` are being used. Based on the document you provided, It seems that passing in all **uppercase letters** should also work normally.